cfgov/paying_for_college/tests/test_models.py

```diff
@@ -77,9 +79,25 @@ def create_page(model, title, slug, parent):
         )
 
     def test_loan_quiz_template(self):
+        page = self.loan_quiz_page
+        stream_block = page.content.stream_block
```

Comment:
Use our existing [`set_stream_data`](https://github.com/cfpb/cfgov-refresh/blob/01a7865e944bc7f1363a818cf58114b598b1f64e/cfgov/v1/util/migrations.py#L90) method to avoid duplication of StreamField surgery?

```py
set_stream_data(page, 'content', [
    {   
        'type': 'guided_quiz',
        'id': '12345',
        'value': {
            'question': '?',                                         
            'answer': 'huh?',
        },
    } 
])       
```

Reply:
good suggestion.
√